backend/src/main/resources/application.properties

```diff
@@ -120,8 +120,8 @@ quarkus.log.cloudwatch.access-key-secret=placeholder
 
 # The current status is cached to limit the number of status DB queries
 quarkus.cache.caffeine.maintenance.expire-after-write=PT60s
-quarkus.cache.caffeine.rbac-recipient-users-provider-get-users.expire-after-write=PT5M
-quarkus.cache.caffeine.rbac-recipient-users-provider-get-group-users.expire-after-write=PT5M
+quarkus.cache.caffeine.rbac-recipient-users-provider-get-users.expire-after-write=PT10M
+quarkus.cache.caffeine.rbac-recipient-users-provider-get-group-users.expire-after-write=PT10M
```

Comment:
As suggested by @pilhuhn on Slack yesterday.

Prod is currently caching data for 1 minute (the 5 minute increase hasn't been deployed).